auslib/blobs/whitelist.py

```diff
@@ -0,0 +1,42 @@
+import re
+from auslib.blobs.base import Blob
+
+import logging
+log = logging.getLogger(__name__)
+
+INDENT = 4 * ' '
+SETTING_TMPL = ('<setting id="%(id)s" '
+                'lastModified="%(last_modified)s"/>')
+
+
+class WhitelistBlobV1(Blob):
+    format_ = {
+        'name': None,
+        'schema_version': None,
+        'whitelist': [
+            { 'imei': None }
+        ]
```

Comment:
Ah, this is an interesting format for the whitelist! In my head I was thinking it would just be a list, but this looks more flexible -- it allows us to whitelist based on multiple parts of the incoming query in the future. Great!